components/contribution-flow/index.js

```diff
@@ -333,7 +333,8 @@ class CreateOrderPage extends React.Component {
 
   /** Validate step profile, create new org if necessary */
   validateStepProfile = async () => {
-    if (!this.state.stepProfile || !this.activeFormRef.current || !this.activeFormRef.current.reportValidity()) {
+    // We don't want to crash if `reportValidity` is not supported by the browser
```

Comment:
Let's remove the comment as it doesn't seems required anymore now that the check is abstracted by the helper 

```suggestion
```

Reply:
yeah correct :)